core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -974,6 +1002,38 @@ private ExprValue exprTimeToSec(ExprValue time) {
     return new ExprLongValue(time.timeValue().toSecondOfDay());
   }
 
+  /**
+   * To_days implementation for ExprValue.
+   *
+   * @return ExprValue.
```

Comment:
JavaDoc should explain what the return is.

Reply:
@Yury-Fridlyand any advice on how to address this if not including "yyyy-MM-dd"

Reply:
You can mention return value type.